### ISSUE TITLE
'com.ouroboros.depends_on' question

### ISSUE THREAD

**Issue by ryester27** [pyouroboros/ouroboros#217]
Hello. I'm reading the description to the "com.ouroboros.depends_on" label, and I can't help but be confused. When I think of dependencies, I think of "something needs to be running or present before I update this container." 

Let's say that container 'torrent' is about to be updated, and is dependent on container 'vpn'. The workflow would be something like this:

1) Ensure that 'vpn' container is up and running
2) Update 'torrent' container

However, this is my interpretation of what "com.ouroboros.depends_on" does, as written in the wiki:

1) Stop 'vpn' container
2) Update 'torrent' container
3) Start 'vpn' container

How "com.ouroboros.depends_on" currently operates is still very useful for me, but I can't help think that "com.ouroboros.depends_on" and "com.ouroboros.hard_depends_on" should be renamed to avoid confusion. Maybe something like "com.ouroboros.restart_after_update" and "com.ouroboros.rebuild_after_update"?

**Comment by DirtyCajunRice** [comment]
we appreciate your feedback, but this is simply named this way to simulate the similarity of docker-compose. As the labels are arbitrary, changing them causes a breaking change for current environments when there is no harm in them being named as they are. If you believe that you can better explain their functions in the wiki i would be more than happy to add any sufficiently explained description.

tl;dr i understand your naming scheme point of view, but as they are already set, the wiki is the best place to describe as opposed to forcing a breaking change for a difference of naming

**Comment by ryester27** [comment]
That makes sense. Looking at the Docker docs, even they use depend_on in a way that I would word differently. It is what it is. Thanks for the explanation!
